package.json

```diff
@@ -95,6 +96,58 @@
 					"description": "A URL for a GitHub that hosts the python.wasm file together with the default Python libraries."
 				}
 			}
-		}
+		},
+		"debuggers": [
+			{
+			  "label": "Python PDB",
+			  "languages": [
+				"python"
+			  ],
+			  "type": "python-pdb-node",
```

Comment:
Added a debug config so I could just test with a launch.json instead of a command.